web/src/dialog_widget.ts

```diff
@@ -133,6 +134,8 @@ export function launch(conf: WidgetConfig): void {
     // * post_render: Callback to run after the modal body is added to DOM.
     // * loading_spinner: Whether to show a loading spinner inside the
     //   submit button when clicked.
+    // * update_submit_on_state_change: If true, updates state of submit button
+    // on valid input change in modal
```

Comment:
The `on` second line should be intended two characters, like in the previous option. Please pay attention to details like this when making changes -- it's your responsibility to write code consistent with the existing style, and it's time-consuming and slows review of your work if one can't count on you to do so.